arch/arm/src/stm32h5/stm32_ethernet.c

```diff
@@ -3604,9 +3604,10 @@ static inline void stm32_ethgpioconfig(struct stm32_ethmac_s *priv)
 #if defined(CONFIG_STM32H5_MII) || defined(CONFIG_STM32H5_RMII)
 
   /* MDC and MDIO are common to both modes */
-
+# ifndef CONFIG_STM32H5_NO_PHY
```

Comment:
where do we define CONFIG_STM32H5_NO_PHY

Reply:
It is defined in `arch/arm/src/stm32h5/Kconfig` 

```
rg ".+_NO_PHY"
arch/arm/src/stm32h7/stm32_ethernet.c
807:#ifndef CONFIG_STM32H7_NO_PHY
3002:#ifndef CONFIG_STM32H7_NO_PHY
3066:#ifndef CONFIG_STM32H7_NO_PHY
3702:# ifndef CONFIG_STM32H7_NO_PHY
4197:#ifdef CONFIG_STM32H7_NO_PHY

arch/arm/src/stm32h7/Kconfig
5545:config STM32H7_NO_PHY

arch/arm/src/stm32h5/stm32_ethernet.c
748:#ifndef CONFIG_STM32H5_NO_PHY
2943:#ifndef CONFIG_STM32H5_NO_PHY
3007:#ifndef CONFIG_STM32H5_NO_PHY
3607:# ifndef CONFIG_STM32H5_NO_PHY
4102:#ifdef CONFIG_STM32H5_NO_PHY

arch/arm/src/stm32h5/Kconfig
4734:config STM32H5_NO_PHY
```